### ISSUE TITLE
check50 not using python3

### ISSUE THREAD

**Issue by adamzu** [hirschhorn/pset6check50#1]
This might just be an issue on my IDE, but when I ran `python check50.py` after all the set-up steps, I got the following error:
```
Traceback (most recent call last):
  File "check50.py", line 26, in <module>
    x, y = MarioLess.test_mario_less()
  File "/home/ubuntu/workspace/pset6check50/check50/MarioLess.py", line 23, in test_mario_less
    process.expect('.*')
  File "/usr/lib/python2.7/dist-packages/pexpect/__init__.py", line 1416, in expect
    compiled_pattern_list = self.compile_pattern_list(pattern)
  File "/usr/lib/python2.7/dist-packages/pexpect/__init__.py", line 1336, in compile_pattern_list
    self._pattern_type_err(p)
  File "/usr/lib/python2.7/dist-packages/pexpect/__init__.py", line 1286, in _pattern_type_err
    for ast in self.allowed_string_types])
TypeError: got <type 'str'> ('.*') as pattern, must be one of: <type 'unicode'>, pexpect.EOF, pexpect.TIMEOUT
```

It looks like this is being caused by the fact that for some reason my IDE is still running python 2 instead of python 3. I was able to get the script to run properly with the command `python3 check50.py`, so the script itself is fine, but is there any way we can enforce in the requirements that python 3 be installed?

**Comment by hirschhorn** [comment]
Have you run udpate50 recently?

**Comment by adamzu** [comment]
Yeah I did. I have 2 c9 accounts, one which I haven't touched in a while, and I was getting errors on that one. Running `update50` didn't seem to fix it.

This is probably a non-issue though because I doubt anyone has an out-of-date account with an out-of-date python setup (I got temporarily locked out of my more recent account, so I was trying it on the old one, but I then figured out my log-in issue with my new account).
